dev/tools/controllerbuilder/template/apis/types.go

```diff
@@ -32,9 +32,6 @@ import (
 
 var {{ .Kind }}GVK = GroupVersion.WithKind("{{ .Kind }}")
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
```

Comment:
A lot of PRs leave this comment in the code. Remove it since it causes more noise than help.

Reply:
+1. Generated code warnings should be kept but not seeing much point in keeping these after a human takes over.